src/components/DatePicker/styles.css

```diff
@@ -0,0 +1,8 @@
+.expensify-datepicker::-webkit-calendar-picker-indicator {
+    position: absolute;
+    right: 12px;
+    top: 13px;
+}
+.expensify-datepicker::-webkit-date-and-time-value {
+    text-align: left;
+}
```

Comment:
Noticed these styles weren't getting applied in storybook so I moved them here - in a `.css` file

Using a `.css` file is a special case here, because it's impossible to target pseudo-elements from js code (e.g. ::-webkit)

It also keeps things together. It was a bit weird for me to have these defined in index.html

Reply:
Please add a code comment explaining this 😄